examples/tensorrt_llm/common/base_engine.py

```diff
@@ -368,3 +406,102 @@ async def async_llm_wrapper():
 
         self._llm_engine = None
         logger.info("Shutdown complete")
+
+    async def _get_remote_prefill_response(self, request):
+        prefill_request = copy.deepcopy(request)
+        prefill_request.sampling_params["max_tokens"] = 1
```

Comment:
nit: if this is the only value changing - maybe we can do a copy of sampling_params only? instead of deepcopy of whole request?

Reply:
It is the value changing. I will investigate it later. But this is how TRTLLM implements disagg in their examples. I wanted to stick to the similar way to avoid issues.
Will try to optimize for sure in future.